#  EXERCICES : LES PRINCIPALES FORMES D'ENERGIES

### _Baptiste LE BRUN - Juline LE BERRE - Rémy VITENNET_


-------------------------------------------------------------------------------------------------------------------------------

##  **_Eexercie 3 :_** Comment dimensionner le moteur d'un ascenseur ?

**Données :**  
- Altitude du 56ème étage par rapport au sol : $d= 196 \, \text{mètres}$
- Temps pour arriver au 56ème étage : $t = 38 \, \text{s}$
- Masse de la cabine d'ascenseur : $m= 5 \, \text{tonnes} = 5000\, \text{kg}$
- Mouvement uniforme 
- Pesanteur : $g= 9.81\; \mathrm{m\cdot s}$

#### **1.**  Calcul de la vitesse de la cabine :

Soit $"v"$ la vitesse de la cabine :  
$$
v= \frac{d}{t}
$$ 


**Calcul :**  
$$
v= \frac{196\; \mathrm{mètres}}{38\; \mathrm{s}}
$$


Soit en formule python : 

In [3]:
d=196
t=38
v=(d)/(t)
v

5.157894736842105

$v= 5.158\; \mathrm{m\cdot s}$

#### **2.**  Calcul de la force nécéssaire pour élever la cabine :

Soit $"F"$ la force nécéssaire pour élever la cabine en Newton :  
$$
F= m \times g
$$ 


**Calcul :**  
$$
F= 5000\; \mathrm{kg} \times 9.81\; \mathrm{m\cdot s}
$$


Soit en formule python : 

In [6]:
m=5000
g=9.81
F=(m)*(g)
F

49050.0

$F= 49050\; \mathrm{N}$

#### **3.**  Calcul de la puisssance utile du moteur :

Soit $"P"$ la puissance mécanique du moteur en Watt :  
$$
P= F \times v
$$ 


**Calcul :**  
$$
P= 49050\; \mathrm{N} \times 5.158\; \mathrm{m\cdot s}
$$


Soit en formule python : 

In [5]:
F=49050
v=5.158
P=(F)*(v)
P

252999.90000000002

$P= 252999\; \mathrm{W}$

-------------------------------------------------------------------------------------------------------------------------------

##  **_Eexercie 4 :_** Comment connaître les paramètres de fonctionnement d'un moteur de voiture ?

**Données :**  
- Vitesse de rotation : $N= 3500\; \mathrm{tr\cdot min} $
- Puissance mécanique : $P = 60 \, \text{kW} = 60000\, \text{W}$
- $360 \, \text{degrée} = 2 \, \mathrm{\pi}  \, \text{rad}$

#### **1.**  Calcul de la vitesse angulaire du moteur :

Soit $"\omega"$ la vitesse angulaire du moteur en rad/s :  
$$
\omega= \frac{N}{60} \times 2\; \mathrm{\pi}
$$ 


**Calcul :**  
$$
\omega= \frac{3500\; \mathrm{tr \cdot min}}{60} \times 2\; \mathrm{\pi}
$$

**D'où :**  
$$
3500\; \mathrm{tr \cdot min}= \frac{3500}{60}\; \mathrm{tr \cdot s}
$$

Soit en formule python : 

In [12]:
N=3500
π=3.14159
w=((N)/60)*2*(π)
w

366.51883333333336

$\omega= 366.51\; \mathrm{rad\cdot s}$

#### **2.**  Calcul du couple que doit développer ce moteur à ce régime moteur :

Soit $"P"$ la puissance mécanique lors d'un mouvement de rotation : 

$$
P= C \times \omega
$$ 

Soit $"C"$ le couple que doit développer ce moteur à ce régime moteur en Newton.mètre : 

$$
C= \frac{P}{\omega}
$$

**Calcul :**  
$$
C= \frac{60000\; \mathrm{W}}{{366.51}\; \mathrm{rad\cdot s}}
$$

Soit en formule python :

In [14]:
P=60000
w=366.51
C=(P)/(w)
C

163.70631087828437

$C= 163.70\; \mathrm{N\cdot m}$

Le moteur doit développer un couple d'environ $163.70\; \mathrm{N\cdot m}$ à une vitesse de 3500 tr.min pour une puissance de 60 kW.

-------------------------------------------------------------------------------------------------------------------------------

##  **_Exercie 5 :_** Comment connaître l'autonomie d'un drone ?

**Données :**  
- Vitesse de rotation trois cellules au lithium associées 
en dérivation et délivrant chacune : $Q = 334\; \mathrm{mA\cdot h} $ sous $U=11,1\; \text{V}$ 
- Puissance du drone : $P=55\; \text{W}$

#### **1.**  Calcul du courant moyen consommé en vol  :

Soit $"P"$ la puissance électrique du drone : 

$$
P= U \times I
$$ 

Soit $"I"$ le l'intensité du courant moyen en ampère: 

$$
I= \frac{P}{\omega}
$$


**Calcul :**  
$$
I= \frac{55\; \mathrm{W}}{11.1\; \mathrm{V}}
$$

Soit en formule python : 

In [15]:
P=55
U=11.1
I=(P)/(U)
I

4.954954954954955

$I= 4.95\; \mathrm{A}$

#### **2.**  Calcul de la capacité totale de la charge électrique disponible :

Soit $"Q_{\text{totale}}"$ la quantité électrique (capacité totale) de la charge electrique disponible en ampère par heure : 

$$
Q_{\text{totale}}= \text{le nombre de batterie} \times Q
$$ 

**Calcul :**  
$$
Q_{\text{totale}}= 3 \times 334\; \mathrm{mA\cdot h}
$$

Soit en formule python : 

In [18]:
n=3
Q=334
Q_totale=(n)*(Q)
Q_totale

1002

$Q_{\text{totale}}=1.002\; \mathrm{A\cdot h}$

#### **3.**  Calcul de l'autonomie totale du drone  :

Soit $"Q_{\text{totale}}"$ la quantité électrique (capacité électrique en en ampère par heure : 

$$
Q_{\text{totale}}= t \times I
$$ 

Soit $"t"$ l'autonomie du drone en heure : 

$$
t= \frac{Q_{\text{totale}}}{I}
$$


**Calcul :**  
$$
t= \frac{1.002\; \mathrm{A\cdot h}}{4.95\; \mathrm{A}}
$$

Soit en formule python : 

In [20]:
Q_totale=1.002
I=4.95
t=(Q_totale)/(I)
t

0.20242424242424242

$t=0.20\; \mathrm{h} = 0.20*60 = 12\; \mathrm{min}$

L'autonomie du drone est donc de 12 minutes.